todoapp/src/components/Todo.tsx

```diff
@@ -168,13 +184,83 @@ const useStyles = makeStyles({
         }
       }
     }
-  }
+  },
 })
 
+
+type Todo = {
+  id: string;
+  text: string;
+  completed: boolean
+}
+
 function Todos(): JSX.Element {
   const isSmallScreen = useMediaQuery('(max-width:600px)');
   const classes = useStyles();
+  const { todosState } = useGlobalState();
+  const [text, setText] = useState("");
+  const [selectedTodoState, setSelectedTodoState] = useState<string | null>(null);
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setText(e.target.value);
+  }
+  const createTodo = (e: FormEvent) => {
+    e.preventDefault();
```

Comment:
don't pass event here, do preventDefault where you're calling the function and then only pass here the required data.